.github/workflows/ci.yml

```diff
@@ -423,25 +423,32 @@ jobs:
           test $TEST_RC -eq 0 || /bin/false
 
   cgroup2:
-    name: CGroups v2 Integration Test
+    name: CGroupsV2 and SELinux Integration
     # nested virtualization is only available on macOS hosts
     runs-on: macos-10.15
     timeout-minutes: 40
     needs: [project, linters, protos, man]
     strategy:
       matrix:
         runc: [runc, crun]
+        # SELinux: replace Permissive with Enforcing after https://github.com/containers/container-selinux/pull/98
+        # is merged and the package becomes generally available.
+        selinux: [Disabled, Permissive]
```

Comment:
Do we need Disabled? Isn't it covered by Ubuntu?

Reply:
I debated this before overwriting the testing that was already being done for cgroupv2. I figured if we incurred failure(s) testing selinux it would be nice to know if the failures also surfaced with selinux disabled in this cgroupv2 context.